RecoParticleFlow/Configuration/python/RecoParticleFlow_cff.py

```diff
@@ -36,11 +36,44 @@
 
 particleFlowLinks = cms.Sequence( particleFlow*particleFlowPtrs*particleBasedIsolationSequence)
 
+from RecoParticleFlow.PFProducer.HGCalTrackCollection_cfi import *
 from RecoParticleFlow.PFProducer.simPFProducer_cfi import *
 from SimTracker.TrackerHitAssociation.tpClusterProducer_cfi import *
 from SimTracker.TrackAssociatorProducers.quickTrackAssociatorByHits_cfi import *
 from Configuration.StandardSequences.Eras import eras
-_phase2_hgcal_simPFSequence = cms.Sequence(tpClusterProducer+quickTrackAssociatorByHits+simPFProducer)
-_phase2_hgcal_particleFlowReco = cms.Sequence( _phase2_hgcal_simPFSequence + particleFlowReco.copy() )
+particleFlowTmpBarrel = particleFlowTmp.clone()
+_phase2_hgcal_particleFlowTmp = cms.EDProducer(
+    "PFCandidateListMerger",
+    src = cms.VInputTag("particleFlowTmpBarrel",
+                        "simPFProducer")
+    
+)
 
+_phase2_hgcal_simPFSequence = cms.Sequence( pfTrack +
+                                            HGCalTrackCollection + 
+                                            tpClusterProducer +
+                                            quickTrackAssociatorByHits +
+                                            simPFProducer )
+_phase2_hgcal_particleFlowReco = cms.Sequence( _phase2_hgcal_simPFSequence*
+                                               particleFlowTrackWithDisplacedVertex*
+                                               pfGsfElectronMVASelectionSequence*
+                                               particleFlowBlock*
+                                               particleFlowEGammaFull*
+                                               particleFlowTmpBarrel*
+                                               particleFlowTmp*
+                                               fixedGridRhoFastjetAllTmp*
+                                               particleFlowTmpPtrs*          
+                                               particleFlowEGammaFinal*
+                                               pfParticleSelectionSequence )
```

Comment:
better put (a copy of) particleFlowReco sequence here without this copy-pasted list of modules/sequences.
Current choice is not well maintainable